### PR TITLE
Update generated client

### LINKED ISSUE #167: Update generated client
I would like to request that you update the generated client for API v1.6.0

### PR DESCRIPTION
**Pull request by mbohlool**
Updating the client after kubernetes 1.6 release. The update process was not trivial as there were breaking changes in OpenAPI spec of kubernetes 1.6 release.

fixes #167 #169

### CODE REVIEW

File: scripts/constants.py [hunk 1]
```diff
@@ -15,15 +15,15 @@
 import sys
 
 # Kubernetes branch to get the OpenAPI spec from.
-KUBERNETES_BRANCH = "release-1.5"
+KUBERNETES_BRANCH = "b0b7a323cc5a4a2019b2e9520c21c7830b7f708e"
```
Comment (dims) [inline scripts/constants.py]: Isn't there a release-1.6 branch now?

**Comment by dims** [comment]
Quick question inline, LGTM 👍

File: scripts/constants.py (hunk 1 above)
Comment (mbohlool) [inline scripts/constants.py]: I don't have a strong opinion about this yet (so I am changing it back to release-1.6) but need need to discuss this. A branch may change (patches) and we should somehow keep track of which spec we created client from (until we have a consistent versioning in spec itself).

**Comment by mbohlool** [comment]
Answered. Thanks.

**Pull request merged**